src/fhirpath/search.py

```diff
@@ -1700,8 +1703,10 @@ def __call__(self, as_json=False):
         self.main_query = self.build()
 
         # TODO handle count with _includes
-        if self.result_params.get("_summary") == "count":
+        if self.result_params.get("_summary") == "count" or self.result_params.get("_count") == 0:
```

Comment:
why did you add this condition here ?

Reply:
I didn't get why I needed to add is neither... I'm not sure why but the test related to `_count=0` was failing because of my changes.
How was this done before? Where did we tell that `_count=0` is equivalent to `_summary=count`?

Reply:
I don't have a fckin clue...